src/pathwayvis/pathwayvis.module.js

```diff
@@ -36,13 +36,15 @@ import { SettingsComponent } from './components/settings/settings.component';
 import { InfoComponent } from './components/info/info.component';
 import { CardControllerComponent } from './components/datacardcontroller/cardcontroller.component'
 import { ReactionsPanelModule } from './components/reactionpanel/reactionspanel.module';
+import { PanelItemComponentModule } from './components/reactionpanel/panelitem/panelitem.module';
 
 export const PathwayVisModule = angular.module('pathwayvis', [
   require('angular-material-data-table'),
   'ngMaterial',
   'ui.router',
   AppModule.name,
   ReactionsPanelModule.name,
```

Comment:
you don't have to add these here. Add it to `metabolica-ui-map/src/pathwayvis/pathwayvis.module.js`

Reply:
it is added in that file

Reply:
right, sorry.